client/app/bundles/course/assessment/question/text-responses.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable */
+// This ArrayInput implementation is only used for the creation of
+// multiple solution words (keywords / lifted words) in the
+// TextResponseComprehensionSolution model,
+// for the purpose of MVP testing and will be replaced with a
+// more intuitive input method (e.g. the tokens for Skills) soon.
+$(document).ready(() => {
```

Comment:
I know this file is temporary but please declare all your variables with `var`, otherwise they will be treated as global variables which could have unintended effects.

Reply:
Have added `var`. Thank you!